vowpalwabbit/core/src/reductions/details/automl/automl_impl.cc

```diff
@@ -396,11 +410,15 @@ void automl<CMType>::offset_learn(multi_learner& base, multi_ex& ec, CB::cb_clas
 
   for (live_slot = 1; static_cast<size_t>(live_slot) < cm->estimators.size(); ++live_slot)
   {
-    if (cm->_lb_trick) { cm->estimators[live_slot].second.update(1, 1 - r); }
+    if (cm->_lb_trick) { cm->estimators[live_slot].second.update(ec[0]->pred.a_s[0].action == labelled_action ? w : 0, 1 - r); }
     else
     {
-      cm->estimators[live_slot].second.update(1, r);
+      cm->estimators[live_slot].second.update(ec[0]->pred.a_s[0].action == labelled_action ? w : 0, r);
```

Comment:
moving this to its own pr https://github.com/VowpalWabbit/vowpal_wabbit/pull/4326